spec/rails_helper.rb

```diff
@@ -62,7 +62,19 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 end
+DatabaseCleaner.strategy = :truncation
```

Comment:
Is everything still working with the db cleaner?

Reply:
yep, everything is working. we weren't setting up the tests correctly yesterday.